src/test_linked_list.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding utf-8 -*-
+import pytest
+
+
+def test_push():
+    from linked_list import LinkedList
+    ll = LinkedList()
+    ll.push('bob')
+    assert ll.head.get_data() == 'bob'
+
```

Comment:
You should also test that when you push onto the list, the size of the list grows.